stagemonitor-core/src/main/java/org/stagemonitor/core/CorePlugin.java

```diff
@@ -396,6 +396,44 @@
 			.configurationCategory(CORE_PLUGIN_NAME)
 			.tags("elasticsearch", "advanced")
 			.buildWithDefault(5);
+	private final ConfigurationOption<Boolean> enableSpringCloudConfigConfigurationSource = ConfigurationOption.booleanOption()
+		.key("stagemonitor.configuration.springcloud.enabled")
+		.dynamic(false)
+		.label("Enable or disable the Spring Cloud Config configuration source. Default is false")
+		.description("Use to let stagemonitor pull configuration from a Spring Cloud Config server via http. It has the lowest precedence." +
+			"{stagemonitor.applicationName} will be used as the application name to fetch configuration for." +
```

Comment:
I'm not sure I like the implicit `{stagemonitor.applicationName}`. Couldn't we just require that this is set in `stagemonitor.configuration.springcloud.address`?

Reply:
Following the idea to only work with full config urls, that does make sense!